Bold CMS from Quintype

```diff
@@ -0,0 +1,84 @@
+{
+    "Timestamp": "2019-09-10'T'12:10:10",
+    "Name": "BoldCMS from Quintype",
+    "Version": "2",
+    "License": "Proprietary",
+    "Inception": "09-2015",
+    "Category": "Essential - Professional",
+    "Open Source": "No",
+    "Cloud Service": "Yes",
+    "On Premises Installation": "No",
+    "Cloud Service Hosted in Europe": "No",
+    "Commercial Support Available?": "Yes",
+    "GraphQL API": "No",
+    "GraphQL [Mutations]": "No",
+    "GraphQL [Subscriptions]": "No",
+    "REST API": "Yes",
+    "REST [Create]": "Yes",
+    "REST [Read]": "Yes",
+    "REST [Update]": "Yes",
+    "REST [Delete]": "Yes",
+    "REST [Upload]": "Yes",
+    "Search API": "Yes",
+    "Search Features [Full Text Search]": "Yes",
+    "Search Features [Stemming]": "No",
+    "Search Features [Stop Words]": "No",
+    "Search Features [Boosting]": "No",
+    "Search Features [Autocompletion]": "No",
+    "Search Features [Autosuggestion]": "Yes",
+    "Search Features [Result Highlighting]": "No",
+    "Search Features [Geospatial Search]": "No",
+    "Search Features [Search within uploads (pdf,doc)]": "No",
+    "Image Manipulation": "Yes",
+    "Image Focalpoint support": "Yes",
+    "Image Facedetection": "No",
+    "Optimized Image Encoding": "Yes",
+    "Asset fingerprinting": "No",
+    "CDN Support": "Yes",
+    "Antivirus Scanning": "No",
+    "Custom Binary Handler": "No",
+    "System Requirements": "NA",
+    "Supported Databases": "NA",
+    "Clustering": "NA",
+    "Docker Support": "NA",
+    "Backup Feature": "Yes",
+    "Import/Export": "Yes",
+    "CLI": "No",
+    "SDK [Java]": "Yes",
+    "SDK [C#]": "No",
+    "SDK [PHP]": "Yes",
+    "SDK [JavaScript]": "Yes",
+    "SDK [React]": "Yes",
+    "SDK [AngularJS]": "Yes",
+    "SDK [TypeScript]": "Yes",
+    "Web Hooks": "Yes",
+    "Eventbus": "No",
+    "Bulk Import": "Yes",
+    "Client Side Forms": "No",
+    "Plugin System": "No",
+    "Customizable UI": "Yes",
+    "Custom Field Types": "Yes",
+    "User Management": "Yes",
+    "Role Based Permissions": "Yes",
+    "Document Level Permissions": "Yes",
+    "OAuth 2.0 Support": "Yes",
+    "Auditing": "Yes",
+    "API Keys": "Yes",
+    "Project Support": "Yes",
+    "I18N Localized Content": "Yes",
+    "Content Trees": "Yes",
+    "Tagging": "Yes",
+    "Content Relations": "1:1, 1:n, n:n, n:1",
+    "Nesting of Fields": "Yes",
+    "GDPR Statement": "No",
+    "GDPR API": "No",
+    "Editing Conflict Detection": "Yes",
+    "Versioning": "Yes",
+    "Content Scheduling": "Yes",
+    "Content Branches": "Yes",
+    "Content Models / Schemas": "Yes",
+    "Content Model / Schema Versioning": "Yes",
+    "Content Migration": "Yes",
+    "Workflows": "Yes",
```

Comment:
Where can I find the documentation on workflows?

Reply:
https://help.quintype.com/platform#release-note-1st-july-2019